Carbonplus-Grasslands/cavan-compact.json

```diff
@@ -0,0 +1,376 @@
+        ],
+        "@context": {
+            "type": "@type",
+            "@vocab": "https://purl.org/geojson/vocab#",
+            "coordinates": {
+                "@container": "@list"
+            }
+        },
+        "geometry": {
+            "type": "Point",
+            "coordinates": [
+                148.9157,
+                -34.8363
+            ]
+        },
+        "relevance": 1,
+        "place_name": "Yass, New South Wales, Australia",
+        "place_type": [
+            "locality"
+        ],
+        "properties": {}
+    },
+    "regen:landOwner": {
+        "@type": "regen:Organization",
+        "schema:email": "Al@macdoch.com",
+        "schema:location": {
+            "feature": {
+                "id": "address.4941533040990382",
+                "text": "Challis Avenue",
+                "type": "Feature",
+                "center": [
+                    151.224816,
+                    -33.869168
+                ],
+                "address": "25",
+                "context": [
+                    {
+                        "id": "postcode.4440734955792810",
+                        "text": "2011"
+                    },
+                    {
+                        "id": "locality.11245911520829820",
+                        "text": "Potts Point",
+                        "wikidata": "Q2918803"
+                    },
+                    {
+                        "id": "place.4960085988742460",
+                        "text": "Sydney",
+                        "wikidata": "Q3130"
+                    },
+                    {
+                        "id": "region.19913417492553050",
+                        "text": "New South Wales",
+                        "wikidata": "Q3224",
+                        "short_code": "AU-NSW"
+                    },
+                    {
+                        "id": "country.9968792518346070",
+                        "text": "Australia",
+                        "wikidata": "Q408",
+                        "short_code": "au"
+                    }
+                ],
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        151.224816,
+                        -33.869168
+                    ]
+                },
+                "relevance": 0.805556,
+                "place_name": "25 Challis Avenue, Potts Point New South Wales 2011, Australia",
+                "place_type": [
+                    "address"
+                ],
+                "properties": {
+                    "accuracy": "point"
+                }
+            },
+            "@context": {
+                "type": "@type",
+                "@vocab": "https://purl.org/geojson/vocab#",
+                "coordinates": {
+                    "@container": "@list"
+                }
+            }
+        },
+        "schema:legalName": "New Kayarem Pty Limited",
+        "schema:name": "New Kayarem Pty Limited",
```

Comment:
this is just a `regen:Organization` so we don't need the `schema:name`
```suggestion
```

Reply:
ditto for all the similar data

Reply:
done